### ISSUE TITLE
Bloodsucker NM Not Spawning correctly

### ISSUE THREAD

**Issue by Mephysto74** [EdenServer/community#3021]
After camping this NM for a while it appears that zone crashes affects it when it should not. I been killing it and was at camp alone at 70h after TOD till 74h and it never spawned. 
It actually spawned once or twice before 70h mark, so this is really confusing and unpredictable.

**Comment by Tjonke** [comment]
I killed this NM 16 times for my ring, it's 60-72h after death for possible spawn.

**Comment by freddiepines** [comment]
It shouldn't be 60-72h, this should be 72 hours EXACT after the last TOD and 21-24 after maintenance (but given the weekly schedule here on Eden it should probably persist like ground kings).  I camped this NM extensively on retail in era - if it's popping in a 12 hour window that's definitely a bug and it needs to be fixed.

![image](https://user-images.githubusercontent.com/50085658/85855937-0aec1980-b785-11ea-8895-4b33dbc54a40.png)

![image](https://user-images.githubusercontent.com/50085658/85855976-1ccdbc80-b785-11ea-9029-d8c1ebb7a6d9.png)

![image](https://user-images.githubusercontent.com/50085658/85856191-906fc980-b785-11ea-82b2-e1db2c1ad214.png)

![image](https://user-images.githubusercontent.com/50085658/85856227-a2516c80-b785-11ea-9a53-f474f6047549.png)

**Comment by bkesecker** [comment]
https://github.com/EdenServer/community/issues/1457

**Comment by Mozzy4Ever** [comment]
Can be closed. Spawns as it should now.

**Comment by bkesecker** [comment]
> Can be closed. Spawns as it should now.

Probably so, but the part that isn't working as described is the shortened window after server maintenance.  There is no code that sets the respawn time to 21-24 hours on cluster start.

**Comment by Mozzy4Ever** [comment]
True. But a lot of NMs don't reset after maintenance like they're supposed to. I suppose with this one specifically it's just extra noticeable due to it typically being 3 day and going to 21-24hr after maintenance

**Comment by Blex64** [comment]
So...it's not spawning as it should, then.

**Comment by danismyhero** [comment]
While I understand wanting to control how many spawns occur due to cluster reboots happening more often compared to retail, the consequence of the persisting TOD is that it's window will never adjust once someone has it on lockdown.

I'm not sure what the perfect solution is for this, but I feel like it should be considered since we are already customizing how it spawns.

**Comment by Mozzy4Ever** [comment]
I think at minimum it'd be good to reset Bloodsucker when we do our server maintenance. Not so applicable to other NMs that are affected by SE server maintenance (like HNMs) as those shift their window naturally due to being a window system. 
Bloodsucker is uniquely affected by the lack of maintenance reset in that it's a long-respawn NM that doesn't have a window, it's instant spawn from ToD. Has resulted in it's ToD only moving (I think...) around 7 or 8 hours later across 2-3 years.

**Comment by Steam-arch** [comment]
Propably a positive thing. Gives people a chance to get the tod without extra nonsense, similar to retail.

Is this a needed feature for the lesser (or even the 2-4 day NMs) in general anymore?

Could maybe free some of the lesser NMs from the list:
Old Two wings
Dobson flies
Colorful Leshy
Centurio XII-I
Shikigami Weapon
Bright-handed Kunberry
Upyri
Lord of Onzozo
Argus
Bune

Hell, could even throw in Tiamat Cerberus and Khimaira to follow their 12-36h (24-72 for tiamat) after maint respawns, or is there a solid argument on why not?

**Comment by KnowOne134** [comment]
I was more talking about if we were to ever clear out the list then it would respawn at a correct time if 21 - 4. Say a new server build, or manually resetting timers. It would not be something we do during our normal mait.

**Comment by KnowOne134** [comment]
As it is now the server doesn't recognize a crash and a mait. So it would still be on the persist timing but if it ever cleared, via a db script or a new server build it would be the 21-24 hours spawning
